### DISCUSSION TITLE
reviewersSampleSize has no effect if bbUseDefaultReviewers=true?

### DISCUSSION THREAD

**Discussion by ablekhman** [renovatebot/renovate#8802] [Request Help] [answered]
We're using hosted instance of Renovate bot to scan repositories on Bitbucket Cloud.

Some repositories have rather long list of default reviewers. When developers create their PRs, they remove some of the default reviewers and leave only relevant ones.

We tried to limit the number of reviewers for Renovate PRs with `reviewersSampleSize` setting. However, it does not have any effect if `bbUseDefaultReviewers=true`.

Is there any way to limit number of reviewers for Bitbucket Cloud repositories with default reviewers specified?

**Comment by rarkins** [comment, marked as answer]
I've checked the code and confirmed it has no effect. The `bbUseDefaultReviewers` option is evaluated at the moment of `createPr()` within the bitbucket platform code, while `reviewersSampleSize` is evaluated after the PR is created and based off any *configured* reviewer list.

As a workaround, you'll need to configure reviewers in Renovate config manually and not use `bbUseDefaultReviewers` if the group is too large.

Alternatively, it will need a feature request issue in this repo for discussion of implementation details, but it would probably need yourself or someone else using BB Cloud with an interest in implementing it.

**Reply by ablekhman** [reply]
Thanks for the prompt answer!

> configure reviewers in Renovate config manually and not use `bbUseDefaultReviewers`

This is what we ebed up doing. We simply added all team members to the `reviewers` setting and disabled `bbUseDefaultReviewers`.
